server/server/session/src/main/java/com/alipay/sofa/registry/server/session/remoting/DataNodeExchanger.java

```diff
@@ -82,19 +82,14 @@ public Response request(Request request) throws RequestException {
                 sessionClient = boltExchange.connect(Exchange.DATA_SERVER_TYPE, url,
                         dataClientHandlers.toArray(new ChannelHandler[dataClientHandlers.size()]));
```

Comment:
To establish a link here, only one link will be established. Is it appropriate to establish a link for the first time? Isn't it necessary to create multiple for the first time?